python-inocybe-openswitch/models/base-etl-common.yang

```diff
@@ -0,0 +1,108 @@
+module base-etl-common {
+
+    namespace "http://www.dellemc.com/networking/os10/dell-base-etl-common";
+
+    prefix base-etl-common;
+
+    organization
+        "Dell EMC";
+
+    contact
+        "http://www.dell.com/support";
+
+    description
+        "This module contains a collection of common YANG definitions for ETL logging and event notification objects.";
+
+    revision 2018-03-30 {
+        description "Revised Version: IETF Compliance";
+        reference "Dell Proprietary";
+    }
+
+    revision 2015-04-21 {
+        description "Initial revision.";
+        reference "Dell Proprietary";
+    }
+
+    typedef severity-level-type {
+        type enumeration {
+            enum "log-emerg" {
+                value "0";
+                description "Emergency level - enum definition starts at 0 due to
+                             required mapping to Linux syslog severity level definitions";
+            }
+            enum "log-alert" {
+                value "1";
```

Comment:
You can put this as a number instead of a string. We have modified this code internally. It would be reflected in the next release.

Reply:
Rakesh, this is a hack model to demonstrate the gaps between what is in OPX and what would be moderately palatable to an upstream developer. It is generated mostly via grep, sed and some minimal editing after that. I do not want to expand any effort on maintaining it. What I would like is to keep it _AS IS_ and ugly _AS IS_ so it clearly demonstrates the gaps until we figure out how do we close these gaps in future releases. Fixing this model by hand will not close them though as they will be reopened on the OPX side straight away.